runtime/decimal.c

```diff
@@ -0,0 +1,17 @@
+#include "balrt.h"
+#include "third-party/decNumber/decQuad.h"
+
+static inline decQuad *balDecToDecQuad(GC decimal128 *dp);
+
+TaggedPtr _bal_decimal_const(const char *decString) {
+    GC decimal128 *dp = _bal_alloc(sizeof(decimal128));
+
+    decContext ctx;
+    decContextDefault(&ctx, DEC_INIT_DECQUAD);
```

Comment:
decQuad/decFloat is documented to only use status and round fields. Eventually we will want to tweak decQuad to use a smaller context object. In the meantime, we should create our own inline function to initialize just these two members.

Reply:
FIxed